scipy/linalg/_decomp_svd.py

```diff
@@ -7,9 +7,15 @@
 from .lapack import get_lapack_funcs, _compute_lwork
 from ._decomp import _asarray_validated
 
+from scipy._lib._array_api import array_namespace, is_numpy
+
 __all__ = ['svd', 'svdvals', 'diagsvd', 'orth', 'subspace_angles', 'null_space']
 
 
+def arg_err_msg(param):
```

Comment:
will be renamed and imported from `_array_api.py` in a follow up to https://github.com/scipy/scipy/pull/19005.